### PR TITLE
delete some redundant code

### PR DESCRIPTION
**Pull request by xichengliudui**
**What type of PR is this?**
 /kind design

**What this PR does / why we need it**:
delete some redundant code

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/apimachinery/pkg/util/net/interface.go [hunk 1]
```diff
@@ -258,9 +258,8 @@ func getIPFromInterface(intfName string, forFamily AddressFamily, nw networkInte
 func memberOf(ip net.IP, family AddressFamily) bool {
 	if ip.To4() != nil {
 		return family == familyIPv4
-	} else {
-		return family == familyIPv6
 	}
+	eturn family == familyIPv6
```
Comment (miguelbernadi) [inline staging/src/k8s.io/apimachinery/pkg/util/net/interface.go]:
```suggestion
	return family == familyIPv6
```

**Comment by miguelbernadi** [comment]
The typo needs fixing. Changes should not affect behaviour of the code as they just remove unnecessary `else` statements.

**Comment by xichengliudui** [comment]
@miguelbernadi ,I ignored it, thank you very much for your remind

**Pull request closed without merging**